bc_obps/common/permissions.py

```diff
@@ -0,0 +1,81 @@
+from typing import Callable, Dict, Literal
+from django.http import HttpRequest
+from registration.utils import raise_401_if_user_not_authorized
+from registration.models import AppRole, UserOperator
+
+all_authorized_app_roles = AppRole.get_all_authorized_app_roles()
+all_industry_user_operator_roles = UserOperator.get_all_industry_user_operator_roles()
+
+permission_configs: Dict[str, Dict] = {
+    "everyone": {
+        'authorized_app_roles': AppRole.get_all_app_roles(),
+        'authorized_user_operator_roles': all_industry_user_operator_roles,
+        'industry_user_must_be_approved': False,
+    },
+    "authorized_roles": {
+        'authorized_app_roles': all_authorized_app_roles,
+        'authorized_user_operator_roles': all_industry_user_operator_roles,
+        'industry_user_must_be_approved': False,
+    },
+    "approved_industry_user": {
+        'authorized_app_roles': ["industry_user"],
+        'authorized_user_operator_roles': all_industry_user_operator_roles,
+    },
+    "industry_user_only": {
+        'authorized_app_roles': ["industry_user"],
+        'authorized_user_operator_roles': all_industry_user_operator_roles,
+        'industry_user_must_be_approved': False,
+    },
+    "approved_industry_admin_user": {
+        'authorized_app_roles': ["industry_user"],
+        'authorized_user_operator_roles': ["admin"],
+    },
+    "authorized_irc_user_only": {
+        'authorized_app_roles': AppRole.get_authorized_irc_roles(),
+    },
+    "approved_authorized_roles": {
+        'authorized_app_roles': all_authorized_app_roles,
+        'authorized_user_operator_roles': all_industry_user_operator_roles,
+    },
+    "authorized_irc_user_and_industry_admin_user": {
+        'authorized_app_roles': all_authorized_app_roles,
+        'authorized_user_operator_roles': ["admin"],
+    },
```

Comment:
Suggest moving this up under `authorized_roles` to keep all the configs that include both cas and industry in the same area